WikipediaLibrary/TWLight
# Add language-adjusted wiki link

## Description
- Added a property to the UserProfile model to determine which Wikipedia TWL page should be linked in the About section.
- Added a management command to update the languages JSON file to check if any new TWL wiki pages have been added/deleted.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
These Wikipedia sites are in the user's preferred language, therefore they might be encouraged to leave comments there

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T299510](https://phabricator.wikimedia.org/T299510)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
1. Navigate to My Library
2. Check that the {lang_code}.wiki link is in your preferred language
3. Change your preferred language in the settings
4. Check that the {lang_code}.wiki link is in that language
5. If a page does not exist in a user's preferred language, then en.wiki shows up by default

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)
![Screenshot 2023-11-17 at 22 13 56](https://github.com/WikipediaLibrary/TWLight/assets/7854953/bf85544e-33c0-40a5-9aaf-4976ae6d6730)
![Screenshot 2023-11-17 at 22 16 18](https://github.com/WikipediaLibrary/TWLight/assets/7854953/43ae9047-0813-49dd-a12e-1841706c1c8b)


## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
